test/test_images/sendeventswithtracing/README.md

```diff
@@ -0,0 +1,13 @@
+# Send Event with Tracing
+
+Send test event with tracing enabled.
```

Comment:
How is this different than using sendevents with the `--add-tracing` flag?

Reply:
It was original version for quick testing. No longer needed. Removed.